signac/core/collection_zarr.py

```diff
@@ -16,36 +18,31 @@ class ZarrCollection(SyncedCollection):
 
     backend = __name__  # type: ignore
 
-    def __init__(self, name=None, store=None, **kwargs):
-        import zarr
+    def __init__(self, group=None, **kwargs):
         import numcodecs  # zarr depends on numcodecs
 
-        self._root = zarr.group(store=store)
-        self._name = name
+        self._root = group
         self._object_codec = numcodecs.JSON()
         super().__init__(**kwargs)
-        if (name is None) == (self._parent is None):
-            raise ValueError(
-                "Illegal argument combination, one of the two arguments, "
-                "parent or name must be None, but not both.")
 
     def _load(self):
-        """Load the data from a Radis-database."""
+        """Load the data."""
         try:
-            dataset = self._root[self._name]
-            data = dataset[0]
+            return self._root[self._name][0]
         except KeyError:
-            data = None
-        return data
+            return None
 
     def _sync(self):
-        """Write the data from Radis-database."""
+        """Write the data."""
         data = self.to_base()
-        # Serialize data:
         dataset = self._root.require_dataset(
             self._name, overwrite=True, shape=1, dtype='object', object_codec=self._object_codec)
         dataset[0] = data
 
+    def __deepcopy__(self, memo):
+        return type(self)(group=self._root, name=self._name, data=self.to_base(),
```

Comment:
For a proper deepcopy, you would need to deepcopy `self._root` and `self._name` as well.